### PR TITLE
MWPW-141047 CI/CD for Akamai edge worker code

### PR DESCRIPTION
**Pull request by TsayAdobe**
- The workflow for deploying an edge worker is `.github/workflows/deploy-edgeworker.yml`
- The code bundles for edge workers are in the directory `edgewokers`
- The unit tests for edge workers are in the directory `test/edgeworkers`
- GitHub Action [Deploy Akamai Edgeworkers](https://github.com/marketplace/actions/deploy-akamai-edgeworkers) is used for deploying to Akamai. This GHA requires scripts other than `main.js` in the `utils` folder. 
- The mocks for unit tests are from [Akamai EdgeWorkers Test Mocks](https://github.com/akamai/edgeworkers-unittest) with some modification
- To run the edge worker unit tests only
  ```
  npm run ewtest
  ```
  The edge worker unit tests are included and the coverage report is generated when running the existing command `npm test` and `npm run jest`.
- To build the edge worker code for the deployment bundle.
  ```
  npm run ewbuild
  ```
  It run the script `edgeworkers/scripts/build.js`, which copies the dependency files to the bundle directory.
- To build the deployment bundle for the stage edge worker.
  ```
  npm run ewprod2stg
  ```
  It run the script `edgeworkers/scripts/prod2stg.js`, which copies the prod edge worker directory to the stage one and modify some flags for the stage environment.  
- To do: Need a script to increment to the version in `bundle.json`
- To do: The code coverage is high, but it may need more unit tests.

### CODE REVIEW

File: edgeworkers/Acrobat_DC_web_prod/bundle.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "edgeworker-version": "0.67",
```
Comment (peller) [inline edgeworkers/Acrobat_DC_web_prod/bundle.json:2]: will we still need to manually update this version number when we deploy?

File: edgeworkers/Acrobat_DC_web_prod/utils/csp/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import stage from './stage.js';
```
Comment (peller) [inline edgeworkers/Acrobat_DC_web_prod/utils/csp/index.js:1]: is there a way we can link everything in the csp/ folder to https://github.com/adobecom/dc/tree/stage/acrobat/scripts/contentSecurityPolicy

**Review by peller — APPROVED** [review]

File: edgeworkers/Acrobat_DC_web_prod/bundle.json (hunk 1 above)
Comment (peller) [reply edgeworkers/Acrobat_DC_web_prod/bundle.json:2]: nevermind.  just saw the answer in the PR comments

File: edgeworkers/scripts/prod2stg.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
```
Comment (peller) [inline edgeworkers/scripts/prod2stg.js:1]: I think what we want is stg2prod?  Typically, we should be promoting changes from stage to prod only. Note: we have a way to detect isProd from the environment without hard-coding it. I'll work on that separately.

**Review by peller — APPROVED** [review]

**Review by sukamat — APPROVED** [review]

**Review by Blainegunn — APPROVED** [review]

File: edgeworkers/scripts/prod2stg.js (hunk 3 above)
Comment (peller) [reply edgeworkers/scripts/prod2stg.js:1]: If you update to the latest edgeworker code on stage, `isProd` is gone. You can remove the code to update that variable.

**Pull request merged**